mmcv/ops/point_sample.py

```diff
@@ -181,25 +255,32 @@ def __init__(self, output_size, spatial_scale, aligned=True):
         self.aligned = aligned
 
     def forward(self, features, rois):
-
         num_imgs = features.size(0)
         num_rois = rois.size(0)
         rel_roi_points = generate_grid(
             num_rois, self.output_size, device=rois.device)
 
-        point_feats = []
-        for batch_ind in range(num_imgs):
-            # unravel batch dim
-            feat = features[batch_ind].unsqueeze(0)
-            inds = (rois[:, 0].long() == batch_ind)
-            if inds.any():
-                rel_img_points = rel_roi_point_to_rel_img_point(
-                    rois[inds], rel_roi_points[inds], feat.shape[2:],
-                    self.spatial_scale).unsqueeze(0)
-                point_feat = point_sample(
-                    feat, rel_img_points, align_corners=not self.aligned)
-                point_feat = point_feat.squeeze(0).transpose(0, 1)
-                point_feats.append(point_feat)
+        if torch.onnx.is_in_onnx_export() and num_imgs == 1:
+            feat = features[0].unsqueeze(0)
```

Comment:
Isn't `features` here is a Tensor of NCHW? It seems `features[0].unsqueeze(0)` makes no sense here.

Reply:
Yes, you are right. Removed.